orcestra_book/plans/HALO-20240909a.md

```diff
@@ -0,0 +1,176 @@
+
+# Load satellite tracks 
+
+ec_fcst_time  = "2024-09-03"
+ec_track = orcestra.sat.SattrackLoader("EARTHCARE", ec_fcst_time, kind="PRE",roi="BARBADOS") \
+    .get_track_for_day(f"{flight_time:%Y-%m-%d}")\
+    .sel(time=slice(f"{flight_time:%Y-%m-%d} 14:00", None))
+
+# Create elements of track
+
+ec_north = point_on_track(ec_track,lat= 16.00).assign(label = "ec_north") 
+ec_south = point_on_track(ec_track,lat=  6.00).assign(label = "ec_south")
+c_north  = point_on_track(ec_track,lat= 12.50).assign(label = "c_north")
+c_south  = point_on_track(ec_track,lat=  8.00).assign(label = "c_south")
+c_mid    = point_on_track(ec_track,lat= 10.25).assign(label = "c_south")
+ec_under = point_on_track(ec_track, lat=14.0, with_time=True).assign(label = "ec_under", note = "meet EarthCARE")
+
+# Additional Waypoints
+
+extra_waypoints = []
+
+# Define Flight Paths
+
+waypoints = [
+     airport.assign(fl=0),
+     ec_north.assign(fl=410),
+     ec_under.assign(fl=410),
+     IntoCircle(c_north.assign(fl=410), radius, 360, enter = 90),   
+     IntoCircle(c_mid.assign(fl=430), radius, 360), 
+     IntoCircle(c_south.assign(fl=450), radius, -360), 
+     ec_south.assign(fl=450),    
+     airport.assign(fl=0),
+     ]
+
+# Crew
+
+crew = {'Mission PI': 'Julia Windmiller',
+        'DropSondes': 'tbd',
+        'HAMP': 'tbd',
+        'SMART/VELOX': 'tbd',
+        'SpecMACS': 'tbd',
+        'WALES' : 'tbd',
+        'Flight Documentation': 'tbd',
+        'Ground Support': 'tbd',
+        }
+
+# Plan
+
+plan = FlightPlan(waypoints, flight_index, extra_waypoints=extra_waypoints, crew=crew)
+print(f"Flight index: {plan.flight_id}")
+print(f"Take-off: {plan.takeoff_time:%H:%M %Z}\nLanding:  {plan.landing_time:%H:%M %Z}\n")
+```
+
+```{code-cell} ipython3
+:tags: [hide-input]
+
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import easygems.healpix as egh
+import intake
+from orcestra.flightplan import plot_cwv, plot_path
+
+forecast_overlay = True
+
+plt.figure(figsize = (14, 8))
+ax = plt.axes(projection=ccrs.PlateCarree())
+ax.set_extent([lon_min, lon_max, lat_min, lat_max], crs=ccrs.PlateCarree())
+ax.coastlines(alpha=1.0)
+ax.gridlines(draw_labels=True, dms=True, x_inline=False, y_inline=False, alpha = 0.25)
+
+if (forecast_overlay):
+    ifs_fcst_time = datetime(2024, 8, 31, 0, 0, 0)
+    ds = intake.open_catalog("https://tcodata.mpimet.mpg.de/internal.yaml").HIFS(datetime = ifs_fcst_time).to_dask().pipe(egh.attach_coords)
+    cwv_flight_time = ds["tcwv"].sel(time=flight_time, method = "nearest")
+    plot_cwv(cwv_flight_time, levels = [48.0, 50.0, 52.0, 54.0, 56.0, 58.0, 60.0], ax=ax)
+    plt.title(f"{flight_time}\n(CWV forecast issued on {ifs_fcst_time})")
+
+plt.plot(ec_track.lon, ec_track.lat, c='k', ls='dotted')
+plot_path(plan, ax, color="C1")
+
+if (False):
+    natal   = LatLon(-5.795,-35.209439, label = "natal")
+    airways = [natal,sal]
+    plot_path(airways, ax, color="purple")
```

Comment:
This doesn't do anything (and it's unlikely that this would tell us anything around Barbados).
```suggestion
```